com.tracktion.Waveform13.yml

```diff
@@ -0,0 +1,136 @@
+id: com.tracktion.Waveform13
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform
+finish-args:
+  - --device=all
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /cache
+  - /include
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '24.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;vst3
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+
+  - name: x264
+    config-opts:
+      - --disable-cli
+      - --enable-shared
+    cleanup:
+      - /include
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: 55d517bc4569272a2c9a367a4106c234aba2ffbc
+
+  - name: ffmpeg
+    config-opts:
+      - --enable-gpl
+      - --enable-shared
+      - --disable-static
+      - --enable-gnutls
+      - --disable-doc
+      - --disable-programs
+      - --disable-devices
+      - --enable-libopus
+      - --enable-libvpx
+      - --enable-libvorbis
+      - --enable-libx264
+    cleanup:
+      - /share/ffmpeg
+      - /include
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-5.1.6.tar.xz
+        sha256: f4fa066278f7a47feab316fef905f4db0d5e9b589451949740f83972b30901bd
+
+  - name: xjadeo
+    cleanup:
+      - /share/man
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/xjadeo/files/xjadeo/v0.8.14/xjadeo-0.8.14.tar.gz
+        sha256: 8f4b0cc7389b1466ce8a9219dd27404b5cbf49590620193930b242a92533d9dd
+
+  - name: xprop
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: 8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753
+
+  - name: libusb
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-1.0.27/libusb-1.0.27.tar.bz2
+        sha256:  ffaa41d741a8a3bee244ac8e54a72ea05bf2879663c098c82fc5757853441575
+
+  - name: waveform
+    buildsystem: simple
+    build-commands:
+      # Icons
+      - install -Dm644 waveform13.png /app/share/icons/hicolor/512x512/apps/waveform13.png
+
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
```

Comment:
once https://github.com/flathub/flathub/pull/5901/files#r1866028324 is done drop this

Reply:
I believe this issue has been fixed